inference-engine/src/cldnn_engine/cldnn_config.h

```diff
@@ -62,6 +62,7 @@ struct Config {
     bool enable_loop_unrolling;
 
     std::map<std::string, std::string> key_config_map;
+    InferenceEngine::PerfHintsConfig  perfHintsConfig;
```

Comment:
why is it dedicated field? not a common / usual config key?

Reply:
This continues the trend of having the common config to stay as dedicated classes.
E.g. StreamsExecutor config, and now PerfHints. This is convenient, as the parsing/error-handling is implemeneted in one place (and not copy-pasted over each plugin), so that e.g. adding a new HINT (like POWER_SAVE) would require adding only to that. From that persp, the key_config_map (above) is only for really plugin-specific items (which e.g. the clDNNPLugin's Config class) handles